src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquirerFactory-GetTokenAcquirers.cs

```diff
@@ -42,49 +42,49 @@ public TokenAcquirerFactory_GetTokenAcquirers(IServiceProvider serviceProvider)
                 };
                 if (region != null)
                 {
-                    microsoftAuthenticationOptions.AzureRegion = region;
+                    MicrosoftIdentityApplicationOptions.AzureRegion = region;
                 }
 
                 var optionsMonitor = ServiceProvider.GetRequiredService<IMergedOptionsStore>();
                 var mergedOptions = optionsMonitor.Get(key);
-                MergedOptions.UpdateMergedOptionsFromMicrosoftAuthenticationOptions(microsoftAuthenticationOptions, mergedOptions);
+                MergedOptions.UpdateMergedOptionsFromMicrosoftIdentityApplicationOptions(MicrosoftIdentityApplicationOptions, mergedOptions);
                 tokenAcquirer = GetTokenAcquirer(key);
             }
             return tokenAcquirer;
         }
 
         /// <inheritdoc/>
-        public ITokenAcquirer GetTokenAcquirer(ApplicationAuthenticationOptions applicationAuthenticationOptions)
+        public ITokenAcquirer GetTokenAcquirer(IdentityApplicationOptions IdentityApplicationOptions)
```

Comment:
>IdentityApplicationOptions [](http://example.com/codeflow?start=47&length=26)

I'm not sure about this name anymore. From a customer perspective, how is this different from MicrosoftIdentityApplicationOptions?

Reply:
Identity = any IdP
MircrosoftIdentity = MicrosoftIdentityPlatform